src/test/java/com/google/jenkins/plugins/k8sengine/KuberentesVerifiersTest.java

```diff
@@ -0,0 +1,83 @@
+package com.google.jenkins.plugins.k8sengine;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.JsonPath;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class KuberentesVerifiersTest {
+  private static final Logger LOGGER = Logger.getLogger(KubernetesVerifiers.class.getName());
+
+  private static final String VERIFIABLE_DEPLOYMENT_OUTPUT = "verifiableDeploymentOutput.json";
+  private static final String UNVERIFIABLE_DEPLOYMENT_OUTPUT = "unverifiableDeploymentOutput.json";
+
+  private static KubectlWrapper kubectl;
```

Comment:
While the objects used in the tests are immutable you may want to consider the work we did on parallelizing the tests and avoid central set up, see #28 for what was done there. The most lightweight example is the ComputeClientTest. However, it might not be necessary given the test class size. I'd be interested in what @craigatgoogle thinks here.

Reply:
Thanks workin on these  - looks like my IDE didn't obey on the wildcard import. Sorry.

Reply:
Changed a bit.